x-pack/plugins/apm/server/lib/transaction_groups/fetcher.ts

```diff
@@ -94,137 +169,47 @@ function getItemsWithRelativeImpact(
         item.sum !== null && item.sum !== undefined
           ? ((item.sum - min) / (max - min)) * 100 || 0
           : 0,
-      p95: item.p95,
```

Comment:
Don't we use p95 anymore?

Reply:
no, the current table doesn't have it. https://github.com/elastic/kibana/issues/91848#issuecomment-885251187